emba.sh

```diff
@@ -188,7 +188,7 @@ main()
   export MAX_EXT_SPACE=11000     # a useful value, could be adjusted if you deal with very big firmware images
   export LOG_DIR="$INVOCATION_PATH""/logs"
   export TMP_DIR="$LOG_DIR""/tmp"
-  export MAIN_LOG_FILE="emba.log"
+  export MAIN_LOG_FILE="emba.txt"
```

Comment:
to we really need to change this? I like the .log for the main log file

Reply:
All other log files are `txt` and it was easier (I am lazy) to change this instead of including `log` files to the webreport.

Reply:
Changed it back to `log`, the webreport can now work with `txt` and `log`